src/compose/service-manager.ts

```diff
@@ -88,7 +88,7 @@ export async function get(service: Service) {
 	// Get the container ids for special network handling
 	const containerIds = await getContainerIdMap(service.appId!);
 	const services = (
-		await getAll(`service-id=${service.serviceId}`)
+		await getAll(`service-name=${service.serviceName}`)
```

Comment:
This is the only one that could be suspect, but it also checks matching appId @20k-ultra so I thin we are I’m the clear